src/backend/index.js

```diff
@@ -70,24 +70,24 @@ function processFeeds(feeds) {
 }
 
 /**
- * Download and parse feed author/URL info from the wiki, and process
+ * Download and parse feed author/URL info from the wiki and redis, and process
  * these into Feed Objects to be added to the database and feed queue.
  */
-async function processWikiFeeds() {
+async function processAllFeeds() {
   try {
-    // Get an Array of Feed objects from the wiki feed list
-    const feeds = await getWikiFeeds();
+    // Get an Array of Feed objects from the wiki feed list and Redis
+    const feeds = Promise.all([...(await Feed.all()), ...(await getWikiFeeds())]);
```

Comment:
Move the `await` out front:

```js
const feeds = await Promise.all([...Feed.all(), ...getWikiFeeds()]);
```

This will be a bit repetitive, since all feeds will include wiki feeds, but we can improve that later.  When we add a feed to the queue, it won't double-add it, so problem solved.

Reply:
Done in https://github.com/Seneca-CDOT/telescope/pull/1015/commits/cd485162fa7f7ec4798be39868c367bc10898133

I'll open this PR to reviews, now, and further suggestions